### PR TITLE
chore: should check deps of all package.json

### PR DESCRIPTION
**Pull request by Timeless0911**
## Summary

Now `check-dependency-version-consistency` only works in `packages/*` folder, we need to cover all of the packages folder.

Since both `changeset` and `check-dependency-version-consistency` will read `workspaces` field in `package.json`, this PR have done such thing:

- Add more packages glob pattern to `workspaces.packages`
- Config `privatePackages` field of `.changeset/config.json` to disable bump private packages
- Modify the test script of `tests/integration/cli/index.test.ts` since after adding `tests/**` to `workspaces.packages` field, `npx rslib inspect` will report an error

```
npm ERR! could not determine executable to run
```

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,11 @@
   "private": true,
   "workspaces": {
     "packages": [
-      "packages/*"
+      "packages/*",
+      "scripts/*",
+      "tests/**",
+      "examples/**",
+      "website"
```
Comment (fi3ework) [inline package.json:10]: Why they're removed, how could they resolve @rslib/core after removed.

**Review by chenjiahan — COMMENTED** [review]
PR Summary pls 👀

File: package.json (hunk 1 above)
Comment (fi3ework) [reply package.json:10]: I was wrong, it's not pnpm workspace file.

**Comment by Timeless0911** [comment]
> PR Summary pls 👀

Just editing before

**Pull request merged**